README.adoc

```diff
@@ -244,12 +276,32 @@ to retrieve the system properties for the specific localhost.
 Notice that the results from the two URLs are identical because the `inventory` service gets its results from
 calling the `system` service.
 
-[role="code_command", subs="quotes"]
+To see the application metrics, point your browser to https://localhost:9443/metrics/application[https://localhost:9443/metrics/application^]. Log in as the `admin` user with `adminpwd` as the password.
+See the following sample outputs for the `@Fallback` annotated method and the fallback method before a fallback occurs:
```

Comment:
Change to:
"The following example shows sample outputs for the `@Fallback` annotated method and the fallback method before a fallback occurs:"

Reply:
The original sentence style was used through out the entire guide. This was to be consistent with the other areas where sample outputs were being displayed.

Reply:
Okay